backendTest.py

```diff
@@ -0,0 +1,24 @@
+import unittest

```

Comment:
`suite = unittest.TestLoader().loadTestsFromTestCase(BackEndTests)`, this basically loads all the functions that start with `test_` into a testsuite which you can then run